codec/reflectcodec/struct_fielder.go

```diff
@@ -65,15 +65,13 @@ type structFielder struct {
 }
 
 func (s *structFielder) GetSerializedFields(t reflect.Type) ([]FieldDesc, error) {
+	if serializedFields, ok := s.getCachedSerializedFields(t); ok { // use pre-computed result
+		return serializedFields, nil
```

Comment:
We expect this to _always_ happen once the node has marshalled / unmarshalled all of the struct types that are registered by the node.